low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -116,6 +116,7 @@ NODEHWSENSOR:
    threaded: true
    polling_interval: 30
    ipmi_client: ipmitool
+   server_declare_frus: ['disk', 'psu', 'fan']
```

Comment:
It would be good if you can use yaml list syntax

Reply:
done